recipe/meta.yaml

```diff
@@ -20,7 +20,7 @@ build:
 requirements:
   host:
     - pip
-    - python >=3.6
+    - python >=3.8
     - setuptools-scm
     - setuptools
   run:
```

Comment:
my understanding of this section is that it should be only the *basic* required runtime dependencies, if someone does `conda install sliderule-python` in a new or existing environment, these are all checked to be present and versions satisfied by the solver, so less is more here. 

since some will run sliderule just via python scripts w/o any widgets i think removing everything but `requests` and `h5py` is the way to go? This might require a little coordination with the sliderule code. If a user calls optional functions like `to_geopandas()` or `ipysliderule.widgets()` and TK or other things are not present in the environment, the client code could raise an error requesting missing packages get installed.